tests/languages/birb/operator_feature.test

```diff
@@ -0,0 +1,27 @@
+++ --
+* / % ~/
++ - ! ~
+<< >> ?

```

Comment:
I suggest to also make `:` an operator to keep the highlighting of `cond ? if_case : else_case` consistent.